### DISCUSSION TITLE
How to read link's parameter

### DISCUSSION THREAD

**Discussion by Creasento** [raisimTech/raisimLib#182] [Q&A] [answered]
I want to read some link's parameters such as pos, vel (of their CoM).

Fortunately, I can read parameters of joint so I make some virtual fixed joints in link CoM and read it.
But I think if can read directly to links, it will be more useful.

Q : Is there an read API such as getFrame~~ for links?

**Comment by jhwangbo** [comment, marked as answer]
The URDF convention does not have a frame attached to a link. There is no link frame but only joint frames. So I cannot define a link velocity...

**Reply by Creasento** [reply]
Thank you for the reply. I'm going to use the present way :)
